<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>315</width>
    <height>424</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>游戏设置</string>
  </property>
  <property name="windowIcon">
   <iconset resource="qtchess.qrc">
    <normaloff>:/images/settings.png</normaloff>:/images/settings.png</iconset>
  </property>
  <widget class="QPushButton" name="btnApply">
   <property name="geometry">
    <rect>
     <x>85</x>
     <y>380</y>
     <width>51</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>应用</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnConfirm">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>380</y>
     <width>51</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>游戏类型：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>谁先走：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>对手执：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>步时：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>180</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>0表示不限时</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>服务器IP：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>本机为：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>服务器端口：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ledStepTime">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>180</y>
     <width>131</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>3000</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ledIpAddr">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>280</y>
     <width>131</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>3000</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ledPort">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>330</y>
     <width>131</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>3000</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbGameType">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>20</y>
     <width>121</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbCompetitorSide">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>70</y>
     <width>81</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbAhead">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>120</y>
     <width>131</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbServerOrClient">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>230</y>
     <width>131</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="qtchess.qrc"/>
 </resources>
 <connections/>
</ui>
